docs/gitbook/tutorials/skipper-progressive-delivery.md

```diff
@@ -0,0 +1,382 @@
+# Skipper Canary Deployments
+
+This guide shows you how to use the [Skipper ingress controller](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/) and Flagger to automate canary deployments.
+
+![Flagger Skipper Ingress Controller](https://raw.githubusercontent.com/weaveworks/flagger/master/docs/diagrams/flagger-skipper-overview.png)
```

Comment:
Is there a flagger-skipper-overview.png?

Reply:
We added this here: https://github.com/weaveworks/flagger/pull/670/files#diff-9864ba2ebb2d970425d71c54634df272

Reply:
Ahh I missed it, all good, thanks